mnefun/_mnefun.py

```diff
@@ -2330,3 +2332,144 @@ def plot_reconstruction(evoked, origin=(0., 0., 0.04)):
     axs[2, 0].set_ylabel('Reconstruction')
     fig.tight_layout()
     return fig
+
+
+def plot_chpi_snr_raw(raw, win_length, n_harmonics):
+    """Compute and plot cHPI SNR from raw data
+
+    Parameters
+    ----------
+    win_length : float
+        Length of window to use for SNR estimates (seconds). A longer window
+        will naturally include more low frequency power, resulting in lower
+        SNR.
+    n_harmonics : int
```

Comment:
Extra 1.609 km?
I'm not 100% sure this makes sense. With 5 kHz it would be dozens of harmonics, majority of which are probably negligible. I can look into it though.